### ISSUE TITLE
Only remove "awaiting change" PR label after all reviewers approved.

### ISSUE THREAD

**Issue by Mariatta** [python/bedevere#71]
When there are multiple reviewers who requested changes, the "awaiting change" label should only be removed after all reviewers approved the new changes.

As brought up by @1st1 in https://github.com/python/cpython/pull/4314:

> core-dev A requests changes; bedevere-bot assigns an awaiting changes label.
> core-dev B approves the PR; bedevere-bot removes the awaiting changes label.
> The label should only be removed when all reviewers clear the PR.

**Comment by brettcannon** [comment]
So I debated with myself about this when I initially implemented the feature. The problem is that core devs are not the most responsive folks in the world due to lack of time. So if two or more people review and just one of them delays, it would then delay the whole PR. This also deviates somewhat from the old workflow where you didn't have to wait for other core devs to sign off if the reviewer clearing the PR thought the other comments were appropriately taken care of.

IOW I don't know if this is a good or bad thing to do, hence why I went with the one that was the easiest to code up. 😁
